WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -58,17 +80,25 @@ - (void)loadView
 
     [WPStyleGuide configureColorsForView:self.view andTableView:self.tableView];
 
-    UINib *headerCellNib = [UINib nibWithNibName:@"CommentTableViewHeaderCell" bundle:nil];
-    UINib *bodyCellNib = [UINib nibWithNibName:@"CommentTableViewCell" bundle:nil];
-    [self.tableView registerNib:headerCellNib forCellReuseIdentifier:CVCHeaderCellIdentifier];
-    [self.tableView registerNib:bodyCellNib forCellReuseIdentifier:CVCCommentCellIdentifier];
-    self.headerLayoutCell = [self.tableView dequeueReusableCellWithIdentifier:CVCHeaderCellIdentifier];
-    self.bodyLayoutCell = [self.tableView dequeueReusableCellWithIdentifier:CVCCommentCellIdentifier];
+    // Register Cell Nibs
+    NSArray *cellClassNames = @[
+        NSStringFromClass([NoteBlockHeaderTableViewCell class]),
+        NSStringFromClass([NoteBlockCommentTableViewCell class]),
+        NSStringFromClass([NoteBlockActionsTableViewCell class])
+    ];
+    
+    for (NSString *cellClassName in cellClassNames) {
+        Class cellClass         = NSClassFromString(cellClassName);
+        NSString *className     = [cellClass classNameWithoutNamespaces];
```

Comment:
Seems like some extra work to stringify the class, just to recreate the class to get another string. 
Would it make sense to do something like `[[NoteBlockHeaderTableViewCell class] classNameWithoutNamespaces]` when creating the `cellClassNames` array?